user/APP/CAN_receive/CAN_receive.c

```diff
@@ -171,31 +171,31 @@ void CAN_CMD_CHASSIS(int16_t motor1, int16_t motor2, int16_t motor3, int16_t mot
 /******************* Public CAN Read Functions *******************/
 
 //Return a pointer to yaw motor data
-const motor_measure_t *get_Yaw_Gimbal_Motor_Measure_Point(void)
+const motor_data_t *get_Yaw_Gimbal_Motor_Measure_Point(void)
```

Comment:
This might break things

Reply:
If we are changing the name of this structure, this needs to be reflected in gimbal_task.h in the struct Gimbal_Motor_t

Reply:
All references to the type were/should have been updated (gimbal_task.h definitely was)